### ISSUE TITLE
Illegal "fi" command in proj4-activate.csh

### ISSUE THREAD

**Issue by paulray** [conda-forge/proj.4-feedstock#87]
Issue:

I found that the last line of proj4-activate.csh (in `mini conda3/envs/hese/etc/conda/activate.d/proj4-activate.csh`) used the bash `fi` command instead of the `endif` that csh is expecting. I fixed that and now my conda activate no longer spits out an error message.  Thought I would report it so it could be fixed here.

<br/>
Environment (<code>conda list</code>):
<details>

```
# packages in environment at /Software/miniconda3/envs/hese:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       0_gnu    conda-forge
alabaster                 0.7.12                     py_0    conda-forge
astropy                   4.0              py37h8f50634_2    conda-forge
attrs                     19.3.0                     py_0    conda-forge
babel                     2.8.0                      py_0    conda-forge
backcall                  0.1.0                      py_0    conda-forge
bleach                    3.1.5              pyh9f0ad1d_0    conda-forge
boost-cpp                 1.72.0               h8e57a91_0    conda-forge
brotlipy                  0.7.0           py37h8f50634_1000    conda-forge
bzip2                     1.0.8                h516909a_2    conda-forge
ca-certificates           2020.4.5.1           hecc5488_0    conda-forge
cairo                     1.16.0            hcf35c78_1003    conda-forge
cartopy                   0.18.0           py37h4b180d9_0    conda-forge
certifi                   2020.4.5.1       py37hc8dfbb8_0    conda-forge
cffi                      1.14.0           py37hd463f26_0    conda-forge
cfgv                      3.1.0                      py_0    conda-forge
cfitsio                   3.470                h3eac812_5    conda-forge
cftime                    1.1.3            py37h03ebfcd_0    conda-forge
chardet                   3.0.4           py37hc8dfbb8_1006    conda-forge
cmarkgfm                  0.4.2            py37h516909a_2    conda-forge
colorama                  0.4.3                      py_0    conda-forge
corner                    2.0.1                   py_1003    conda-forge
coverage                  5.1              py37h8f50634_0    conda-forge
coveralls                 2.0.0                      py_0    conda-forge
cryptography              2.9.2            py37hb09aad4_0    conda-forge
curl                      7.69.1               h33f0ec9_0    conda-forge
cycler                    0.10.0                     py_2    conda-forge
dbus                      1.13.6               he372182_0    conda-forge
decorator                 4.4.2                      py_0    conda-forge
defusedxml                0.6.0                      py_0    conda-forge
docopt                    0.6.2                      py_1    conda-forge
docutils                  0.16             py37hc8dfbb8_1    conda-forge
dspsr                     2020.04.03       py37h33ec4a3_1    demorest
editdistance              0.5.3            py37h3340039_0    conda-forge
emcee                     3.0.2              pyh5ca1d4c_0    conda-forge
entrypoints               0.3             py37hc8dfbb8_1001    conda-forge
epsic                     2020.01.03           h6bb024c_1    demorest
expat                     2.2.9                he1b5a44_2    conda-forge
fancycompleter            0.8                   py37_1000    conda-forge
fftw                      3.3.8           nompi_h7f3a6c3_1110    conda-forge
filelock                  3.0.10                     py_0    conda-forge
flake8                    3.7.9            py37hc8dfbb8_1    conda-forge
fontconfig                2.13.1            h86ecdb6_1001    conda-forge
freetype                  2.10.1               he06d7ca_0    conda-forge
freexl                    1.0.5             h14c3975_1002    conda-forge
future                    0.18.2           py37hc8dfbb8_1    conda-forge
gdal                      3.0.4            py37h4b180d9_6    conda-forge
geos                      3.8.1                he1b5a44_0    conda-forge
geotiff                   1.5.1               h05acad5_10    conda-forge
gettext                   0.19.8.1          hc5be6a0_1002    conda-forge
giflib                    5.2.1                h516909a_2    conda-forge
gitdb                     4.0.5                      py_0    conda-forge
gitpython                 3.1.2                      py_0    conda-forge
glib                      2.64.2               h6f030ca_0    conda-forge
gsl                       2.6                  h294904e_0    conda-forge
gst-plugins-base          1.14.5               h0935bb2_2    conda-forge
gstreamer                 1.14.5               h36ae1b5_2    conda-forge
h5py                      2.10.0          nompi_py37h513d04c_102    conda-forge
hdf4                      4.2.13            hf30be14_1003    conda-forge
hdf5                      1.10.5          nompi_h3c11f04_1104    conda-forge
hendrics                  4.0rc1.post2          py37_1001    conda-forge
hypothesis                5.12.0                     py_0    conda-forge
icu                       64.2                 he1b5a44_1    conda-forge
identify                  1.4.15             pyh9f0ad1d_0    conda-forge
idna                      2.9                        py_1    conda-forge
imagesize                 1.2.0                      py_0    conda-forge
importlib-metadata        1.6.0            py37hc8dfbb8_0    conda-forge
importlib_metadata        1.6.0                         0    conda-forge
ipykernel                 5.2.1            py37h43977f1_0    conda-forge
ipython                   7.14.0           py37hc8dfbb8_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
jedi                      0.17.0           py37hc8dfbb8_0    conda-forge
jeepney                   0.4.3                      py_0    conda-forge
jinja2                    2.11.2             pyh9f0ad1d_0    conda-forge
jpeg                      9c                h14c3975_1001    conda-forge
jplephem                  2.14               pyhe916df7_1    conda-forge
json-c                    0.13.1            h14c3975_1001    conda-forge
jsonschema                3.2.0            py37hc8dfbb8_1    conda-forge
jupyter_client            6.1.3                      py_0    conda-forge
jupyter_core              4.6.3            py37hc8dfbb8_1    conda-forge
jupytext                  1.4.2              pyh9f0ad1d_0    conda-forge
kealib                    1.4.13               hec59c27_0    conda-forge
keyring                   21.2.1           py37hc8dfbb8_0    conda-forge
kiwisolver                1.2.0            py37h99015e2_0    conda-forge
krb5                      1.17.1               h2fd8d38_0    conda-forge
ld_impl_linux-64          2.34                 h53a641e_0    conda-forge
libblas                   3.8.0               14_openblas    conda-forge
libcblas                  3.8.0               14_openblas    conda-forge
libclang                  9.0.1           default_hde54327_0    conda-forge
libcurl                   7.69.1               hf7181ac_0    conda-forge
libdap4                   3.20.6               h1d1bd15_0    conda-forge
libedit                   3.1.20170329      hf8c457e_1001    conda-forge
libffi                    3.2.1             he1b5a44_1007    conda-forge
libgcc-ng                 9.2.0                h24d8f2e_2    conda-forge
libgdal                   3.0.4                h3dfc09a_6    conda-forge
libgfortran-ng            7.3.0                hdf63c60_5    conda-forge
libgomp                   9.2.0                h24d8f2e_2    conda-forge
libiconv                  1.15              h516909a_1006    conda-forge
libkml                    1.3.0             hb574062_1011    conda-forge
liblapack                 3.8.0               14_openblas    conda-forge
libllvm8                  8.0.1                hc9558a2_0    conda-forge
libllvm9                  9.0.1                he513fc3_1    conda-forge
libnetcdf                 4.7.4           nompi_h9f9fd6a_101    conda-forge
libopenblas               0.3.7                h5ec1e0e_6    conda-forge
libpng                    1.6.37               hed695b0_1    conda-forge
libpq                     12.2                 h5513abc_1    conda-forge
libsodium                 1.0.17               h516909a_0    conda-forge
libspatialite             4.3.0a            h2482549_1038    conda-forge
libssh2                   1.8.2                h22169c7_2    conda-forge
libstdcxx-ng              9.2.0                hdf63c60_2    conda-forge
libtiff                   4.1.0                hc7e4089_6    conda-forge
libuuid                   2.32.1            h14c3975_1000    conda-forge
libwebp-base              1.1.0                h516909a_3    conda-forge
libxcb                    1.13              h14c3975_1002    conda-forge
libxkbcommon              0.10.0               he1b5a44_0    conda-forge
libxml2                   2.9.10               hee79883_0    conda-forge
llvmlite                  0.31.0           py37h5202443_1    conda-forge
lz4-c                     1.9.2                he1b5a44_1    conda-forge
markupsafe                1.1.1            py37h8f50634_1    conda-forge
matplotlib                3.2.1                         0    conda-forge
matplotlib-base           3.2.1            py37h30547a4_0    conda-forge
mccabe                    0.6.1                      py_1    conda-forge
mistune                   0.8.4           py37h8f50634_1001    conda-forge
more-itertools            8.2.0                      py_0    conda-forge
mpi                       1.0                     openmpi    conda-forge
mpi4py                    3.0.3            py37hbfacf26_1    conda-forge
nbconvert                 5.6.1            py37hc8dfbb8_1    conda-forge
nbdime                    2.0.0                      py_1    conda-forge
nbformat                  5.0.6                      py_0    conda-forge
nbsphinx                  0.7.0              pyh9f0ad1d_0    conda-forge
ncurses                   6.1               hf484d3e_1002    conda-forge
netcdf4                   1.5.3           nompi_py37hec16513_103    conda-forge
nodeenv                   1.3.5                      py_0    conda-forge
notebook                  6.0.3            py37hc8dfbb8_0    conda-forge
nspr                      4.25                 he1b5a44_0    conda-forge
nss                       3.47                 he751ad9_0    conda-forge
numba                     0.48.0           py37hb3f55d8_0    conda-forge
numpy                     1.18.4           py37h8960a57_0    conda-forge
olefile                   0.46                       py_0    conda-forge
openjpeg                  2.3.1                h981e76c_3    conda-forge
openmpi                   4.0.3                hdf1f1ad_1    conda-forge
openssl                   1.1.1g               h516909a_0    conda-forge
owslib                    0.19.2                     py_1    conda-forge
packaging                 20.1                       py_0    conda-forge
pandoc                    2.9.2.1                       0    conda-forge
pandocfilters             1.4.2                      py_1    conda-forge
parso                     0.7.0              pyh9f0ad1d_0    conda-forge
pcre                      8.44                 he1b5a44_0    conda-forge
pdbpp                     0.10.0                     py_0    conda-forge
pexpect                   4.8.0            py37hc8dfbb8_1    conda-forge
pgplot                    5.2.2             hac706ba_1006    conda-forge
pickleshare               0.7.5           py37hc8dfbb8_1001    conda-forge
pillow                    7.1.2            py37h718be6c_0    conda-forge
pint-pulsar               0.6.3            py37hc8dfbb8_0    conda-forge
pip                       20.1               pyh9f0ad1d_0    conda-forge
pixman                    0.38.0            h516909a_1003    conda-forge
pkginfo                   1.5.0.1                    py_0    conda-forge
pluggy                    0.13.1           py37hc8dfbb8_1    conda-forge
poppler                   0.67.0               h14e79db_8    conda-forge
poppler-data              0.4.9                         1    conda-forge
postgresql                12.2                 h8573dbc_1    conda-forge
pre-commit                2.3.0            py37hc8dfbb8_0    conda-forge
proj                      7.0.0                h966b41f_3    conda-forge
prometheus_client         0.7.1                      py_0    conda-forge
prompt-toolkit            3.0.5                      py_0    conda-forge
psrchive                  2020.05.08       py37h33ec4a3_1    demorest
pthread-stubs             0.4               h14c3975_1001    conda-forge
ptyprocess                0.6.0                   py_1001    conda-forge
py                        1.8.1                      py_0    conda-forge
pycodestyle               2.5.0                      py_0    conda-forge
pycparser                 2.20                       py_0    conda-forge
pyepsg                    0.4.0                      py_0    conda-forge
pyflakes                  2.1.1                      py_0    conda-forge
pygments                  2.6.1                      py_0    conda-forge
pyopenssl                 19.1.0                     py_1    conda-forge
pyparsing                 2.4.7              pyh9f0ad1d_0    conda-forge
pyproj                    2.6.1.post1      py37h34dd122_0    conda-forge
pyqt                      5.12.3           py37h8685d9f_3    conda-forge
pyqt5-sip                 4.19.18                  pypi_0    pypi
pyqtchart                 5.12                     pypi_0    pypi
pyqtwebengine             5.12.1                   pypi_0    pypi
pyrsistent                0.16.0           py37h8f50634_0    conda-forge
pyshp                     2.1.0                      py_0    conda-forge
pysocks                   1.7.1            py37hc8dfbb8_1    conda-forge
pytest                    5.4.2            py37hc8dfbb8_0    conda-forge
python                    3.7.6           h8356626_5_cpython    conda-forge
python-dateutil           2.8.1                      py_0    conda-forge
python_abi                3.7                     1_cp37m    conda-forge
pytz                      2020.1             pyh9f0ad1d_0    conda-forge
pyyaml                    5.3.1            py37h8f50634_0    conda-forge
pyzmq                     19.0.1           py37hac76be4_0    conda-forge
qt                        5.12.5               hd8c4c69_1    conda-forge
readline                  8.0                  hf8c457e_0    conda-forge
readme_renderer           24.0                       py_0    conda-forge
requests                  2.23.0             pyh8c360ce_2    conda-forge
requests-toolbelt         0.9.1                      py_0    conda-forge
schwimmbad                0.3.1            py37hc8dfbb8_0    conda-forge
scipy                     1.4.1            py37ha3d9a3c_3    conda-forge
secretstorage             3.1.2            py37hc8dfbb8_1    conda-forge
send2trash                1.5.0                      py_0    conda-forge
setuptools                46.2.0           py37hc8dfbb8_0    conda-forge
shapely                   1.7.0            py37hc88ce51_3    conda-forge
six                       1.14.0                     py_1    conda-forge
smmap                     3.0.4              pyh9f0ad1d_0    conda-forge
snowballstemmer           2.0.0                      py_0    conda-forge
sortedcontainers          2.1.0                      py_0    conda-forge
sphinx                    3.0.3                      py_0    conda-forge
sphinx_rtd_theme          0.4.3                      py_0    conda-forge
sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    1.0.3                      py_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.4                      py_0    conda-forge
sqlite                    3.30.1               hcee41ef_0    conda-forge
stingray                  0.1              py37h5ca1d4c_0    conda-forge
tbb                       2018.0.5             h2d50403_0    conda-forge
tempo                     2020.03.26           h9ac9557_1    demorest
tempo-clockfiles          2020.05.07                    1    demorest
tempo2                    2019.01.1            h1c2f66e_1    demorest
terminado                 0.8.3            py37hc8dfbb8_1    conda-forge
testpath                  0.4.4                      py_0    conda-forge
tiledb                    1.7.7                h8efa9f0_1    conda-forge
tk                        8.6.10               hed695b0_0    conda-forge
toml                      0.10.0                     py_0    conda-forge
tornado                   6.0.4            py37h8f50634_1    conda-forge
tox                       3.15.0           py37hc8dfbb8_0    conda-forge
tqdm                      4.46.0             pyh9f0ad1d_0    conda-forge
traitlets                 4.3.3            py37hc8dfbb8_1    conda-forge
twine                     3.1.1                    py37_0    conda-forge
tzcode                    2020a                h516909a_0    conda-forge
uncertainties             3.1.2                    py37_0    conda-forge
urllib3                   1.25.9                     py_0    conda-forge
virtualenv                16.7.5                     py_0    conda-forge
wcwidth                   0.1.9              pyh9f0ad1d_0    conda-forge
webencodings              0.5.1                      py_1    conda-forge
wheel                     0.34.2                     py_1    conda-forge
wmctrl                    0.3             py37hc8dfbb8_1002    conda-forge
xerces-c                  3.2.2             h8412b87_1004    conda-forge
xorg-kbproto              1.0.7             h14c3975_1002    conda-forge
xorg-libice               1.0.10               h516909a_0    conda-forge
xorg-libsm                1.2.3             h84519dc_1000    conda-forge
xorg-libx11               1.6.9                h516909a_0    conda-forge
xorg-libxau               1.0.9                h14c3975_0    conda-forge
xorg-libxdmcp             1.1.3                h516909a_0    conda-forge
xorg-libxext              1.3.4                h516909a_0    conda-forge
xorg-libxrender           0.9.10            h516909a_1002    conda-forge
xorg-renderproto          0.11.1            h14c3975_1002    conda-forge
xorg-xextproto            7.3.0             h14c3975_1002    conda-forge
xorg-xproto               7.0.31            h14c3975_1007    conda-forge
xz                        5.2.5                h516909a_0    conda-forge
yaml                      0.2.4                h516909a_0    conda-forge
zeromq                    4.3.2                he1b5a44_2    conda-forge
zipp                      3.1.0                      py_0    conda-forge
zlib                      1.2.11            h516909a_1006    conda-forge
zstd                      1.4.4                h6597ccf_3    conda-forge

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```

     active environment : hese
    active env location : /Software/miniconda3/envs/hese
            shell level : 1
       user config file : /home/paulr/.condarc
 populated config files : /home/paulr/.condarc
          conda version : 4.8.3
    conda-build version : not installed
         python version : 3.7.6.final.0
       virtual packages : __glibc=2.17
       base environment : /Software/miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /Software/miniconda3/pkgs
                          /home/paulr/.conda/pkgs
       envs directories : /Software/miniconda3/envs
                          /home/paulr/.conda/envs
               platform : linux-64
             user-agent : conda/4.8.3 requests/2.23.0 CPython/3.7.6 Linux/3.10.0-1127.el7.x86_64 centos/7.8.2003 glibc/2.17
                UID:GID : 7655:7650
             netrc file : None
           offline mode : False


```
</details>

**Comment by xylar** [comment]
Thanks for reporting this @paulray.  I just fixed it yesterday in #86.  It will be a little while (days to weeks) before this new version (7.0.1) becomes the new default on `conda-forge` but that that point this issue should be fixed.

If you're good with that, I'll close this issue.  If you feel we should also build 7.0.0 with this fix, I can do that.

**Comment by paulray** [comment]
Great, thanks! Fine to close now.
